pkg/admission/apibinding/binding_permissions.go

```diff
@@ -26,9 +26,21 @@ import (
 	apisv1alpha1 "github.com/kcp-dev/kcp/sdk/apis/apis/v1alpha1"
 )
 
-func CheckAPIExportAccess(ctx context.Context, user user.Info, apiExportName string, authz authorizer.Authorizer) error {
+func CheckAPIExportAccess(ctx context.Context, u user.Info, apiExportName string, authz authorizer.Authorizer) error {
+	groups := u.GetGroups()
+	if sourceCluster, ok := u.GetExtra()["authentication.kubernetes.io/cluster-name"]; ok {
+		groups = append(groups, "system:cluster:"+sourceCluster[0])
```

Comment:
```suggestion
		for _, clusterName := range sourceCluster {
			groups = append(groups, "system:cluster:"+clusterName)
		}
```
Just in case and keeps with the style of other places where these attributes where accessed.